atlascope/core/models/detected_nucleus.py

```diff
@@ -112,12 +118,29 @@ class DetectedNucleus(models.Model):
     )
 
 
+def similar_nuclei(nucleus, queryset=None):
+    """Return a queryset of nuclei ordered by similarity."""
+    if queryset is None:
+        queryset = DetectedNucleus.objects.all()
+    return queryset.annotate(
+        dissimilarity=CubeDistance('fingerprint', nucleus.fingerprint)
+    ).order_by('dissimilarity')
+
+
 class DetectedNucleusSerializer(serializers.ModelSerializer):
+    fingerprint = serializers.ListField(
```

Comment:
Will we use this field for client side visualization? If not, it is a large amount of data to transfer and we'll need a reason to include it in the serializer

Reply:
I had planned on using this for client side visualization, but on second thought I think not. I'll drop this serializer in preference for a `/nucleus-detections/{id}/fingerprint` endpoint that encodes this fingerprint into an image server-side.

Reply:
Fixed in 83fd6218d62fa3db4967f7c51ef576e1dc4ae5ee and 63666378135dae79335ca55d34396624f652b686.